sunpy/physics/limb_darkening.py

```diff
@@ -0,0 +1,90 @@
+import numpy as np
+from astropy import units as u
+from astropy.coordinates import SkyCoord
+
+
+def limbdark(observation, limb_cut=True):
```

Comment:
This remove the limbdarkening effect... I'm not sure whether this is the best name for the function.